src/App.tsx

```diff
@@ -1,14 +1,85 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [openedTodo, setOpenedTodo] = useState<boolean>(false);
+  const [todoInfo, setTodoInfo] = useState<Todo | null>(null);
+  const [openedTodoId, setOpenedTodoId] = useState<number | null>(null);
+  const [filterOption, setFilterOption] = useState<boolean | null>(null);
```

Comment:
it's easy to set filter option as boolean if you have only 2 options)
but it's better to store them as a string in enum - it will be easier to extend number of options